src/main/kotlin/FunCommands.kt

```diff
@@ -40,6 +45,10 @@ class FunCommands(
         val commands = Json.decodeFromString<List<FunCommandConfig>>(commandsJson)
         logger.info("Parsed ${commands.size} commands from JSON.")
 
+        VelocityCommandManager(proxy, chattORE).apply {
```

Comment:
I'd rather this command be registered where all other commands are registered, inside of `ChattORE.kt`.